.github/workflows/main.yml

```diff
@@ -23,9 +23,14 @@ jobs:
           cp -r ./build/chrome ./extensions
           cp -r ./build/edge ./extensions
           cp -r ./build/firefox ./extensions
-          
         env:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - name: bumping version
+        uses: 'phips28/gh-action-bump-version@master'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          version-type: 'minor'
```

Comment:
should we be defaulting to minor? this should be using the built in patterns right?